test/automation/navigation.tests.ps1

```diff
@@ -1209,7 +1209,135 @@ AfterEach{
         test-path .\Bill\BITS | Should be $True
         test-path .\Bill\BITS -Type Container | Should be $False
         test-path .\Bill\BITS -Type Leaf | Should be $True
-    }
+     }
        
  }
+
+Describe "Not Supported Commands test" -Tags "Feature"{
+BeforeEach{
+        cd $home
+        $a=Get-PSDrive -PSProvider SHiPS
+        $a | % {remove-psdrive $_.Name -ErrorAction Ignore}
+
+        $a = new-psdrive -name ss -psprovider SHiPS -root test#Root
+        $a.Name | Should Be "ss"
+
+        cd $testpath
+       }
+
+AfterEach{
+        cd $home
+        $a=Get-PSDrive -PSProvider SHiPS
+        $a | % {remove-psdrive $_.Name -ErrorAction Ignore}
+
+        If(Test-Path $script:homePath) {Remove-Item -Path $script:homePath -force -Recurse -ErrorAction Ignore}
+        
```

Comment:
My comments from the first PR still apply to these tests.

Reply:
Fixed.